.github/workflows/ci.yaml

```diff
@@ -1,17 +1,43 @@
 name: CI Pipeline
 
 on:
+  workflow_call:
+    secrets:
+      MAIL_USERNAME:
+        description: 'Username for the email notification step'
+        required: true
+      MAIL_PASSWORD:
+        description: 'Password for the email notification step'
+        required: true
+    inputs:
+      custom_docker_tag:
+        description: 'Custom Docker tag'
+        # If not specified, then the 'latest' tag will be used
+        default: ''
+        required: false
+        type: string
+
   push:
     branches:
       - master
   pull_request:
     branches:
       - master
 
+env:
+  DOCKER_REGISTRY: ghcr.io
+  DOCKER_REPO: angular-react-starter-image
+  DOCKER_TAG: ${{ inputs.custom_docker_tag || 'latest' }}
```

Comment:
LEt's not add such a var, let's pass a default value to input instead

Reply:
Done